tests/end-to-end/upgrade/pkg/tests/eventmesh-migration/helpers/event.go

```diff
@@ -0,0 +1,261 @@
+package helpers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"k8s.io/apimachinery/pkg/labels"
+
+	apiv1 "github.com/kyma-project/kyma/components/event-bus/apis/eventing/v1alpha1"
+
+	subApis "github.com/kyma-project/kyma/components/event-bus/apis/eventing/v1alpha1"
+	ebClientSet "github.com/kyma-project/kyma/components/event-bus/client/generated/clientset/internalclientset"
+
+	"github.com/avast/retry-go"
```

Comment:
Not sure if I understand the logic behind this packet grouping.

Reply:
fixed in other PR... this one was a duplicate